capstone/fitness/src/main/resources/client_secret.json

```diff
@@ -0,0 +1 @@
+{"web":{"client_id":"75432476031-d2v8phdfhuq64hsrjhuc5q4thiu0nblh.apps.googleusercontent.com","project_id":"piercedw-step-2020","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"eYEPWdpCc6hCQBomCM3r1B8p","redirect_uris":["http://localhost:8080/oauth2callback","https://8080-b7796e94-2ff5-4c18-b6c4-2c972cdc0627.us-central1.cloudshell.dev/oauth2callback","https://8080-b7796e94-2ff5-4c18-b6c4-2c972cdc0627.us-central1.cloudshell.dev/calendar-handler"]}}
```

Comment:
never commit credentials in github. talk to @GabrielGonzalezDiaz about how to set credentials, eg. env variables.

Reply:
Okay. Will talk to Gabriel tmr.